cloud_experts_tutorials/cloud-experts-getting-started/cloud-experts-getting-started-managing-worker-nodes.adoc

```diff
@@ -0,0 +1,238 @@
++
+[source,terminal]
+----
+rosa list machinepools --cluster=<cluster-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID          AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS            TAINTS    AVAILABILITY ZONES
+Default     No           2         m5.xlarge                                  us-east-1a
+----
+
+=== Creating a machine pool with the UI
+. Log in to the link:https://console.redhat.com/openshift[Red Hat console] and click your cluster.
++
+image::cloud-experts-getting-started-managing-ocm-cluster.png[]
+
+. Click *Machine pools*.
++
+image:cloud-experts-getting-started-managing-mp-ocm.png[]
+
+. Click *Add machine pool*.
+
+. Enter the desired configuration.
++
+[TIP]
+====
+You can also and expand the *Edit node labels and taints* section to add node labels and taints to the nodes in the machine pool.
+====
++
+image::cloud-experts-getting-started-managing-mp-nlt.png[]
+
+. You will see the new machine pool you created.
++
+image::cloud-experts-getting-started-managing-mp-fromui.png[]
+
+== Scaling worker nodes
+
+Edit a machine pool to scale the number of worker nodes in that specific machine pool. You can use either the CLI or the UI to scale worker nodes.
+
+=== Scaling worker nodes using the CLI
+
+. Run the following command to see the default machine pool that is created with each cluster:
++
+[source,terminal]
+----
+rosa list machinepools --cluster=<cluster-name>
+----
++
+.Example output
++
+[source,terminal]
+----
+ID          AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS            TAINTS    AVAILABILITY ZONES
+Default     No           2         m5.xlarge                                  us-east-1a
+----
+
+. To scale the default machine pool out to 3 nodes, run the following command:
++
+[source,terminal]
+----
+rosa edit machinepool --cluster=<cluster-name> --replicas=<number-nodes> <machinepool-name>
+----
++
+.Example input
++
+[source,terminal]
+----
+rosa edit machinepool --cluster=my-rosa-cluster --replicas 3 Default
+----
+
+. Run the following command to confirm that the machine pool has scaled:
++
+[source,terminal]
+----
+rosa describe cluster --cluster=<cluster-name> | grep Compute
+----
++
+.Example input
++
+[source,terminal]
+----
+$ rosa describe cluster --cluster=my-rosa-cluster | grep Compute
+----
++
+.Example output
++
+[source,terminal]
+----
+- Compute:                 3 (m5.xlarge)
+----
+
+=== Scaling worker nodes using the UI
+
+. Click the three dots to the right of the machine pool you want to edit.
+. Click *Edit*.
+. Enter the desired number of nodes, and click *Save*.
+. Confirm that the cluster has scaled by selecting the cluster, clicking the *Overview* tab, and scrolling to *Compute listing*. The compute listing should equal the scaled nodes. For example, 3/3.
++
+image::cloud-experts-getting-started-managing-ocm-nodes.png[]
+
+=== Adding node labels
+
+[NOTE]
+====
+Labels and taints cannot be added to the default machine pool.
```

Comment:
This is actually not the case anymore. It is (unfortunately) a bit complicated.  It may make sense to remove this note completely, as you point to the more descriptive docs earlier in this doc...maybe? Regardless, I think this note needs to be removed or updated. https://docs.openshift.com/rosa/rosa_cluster_admin/rosa_nodes/rosa-managing-worker-nodes.html#rosa-adding-taints-clirosa-managing-worker-nodes. 
![image](https://github.com/openshift/openshift-docs/assets/122639474/64654846-bdda-4f1a-9780-7ded06cb37b5)